src/components/ReleaseService/ReleasePlan/TriggerRelease/AddIssueSection/CVEFormContent.tsx

```diff
@@ -42,23 +40,11 @@ const CVEFormContent: React.FC<CVEFormContentProps> = ({ modalToggle }) => {
           </TextContent>
         </StackItem>
         <StackItem>
-          <InputField data-test="cve-issue-key" label="CVE key" name="key" required />
+          <InputField data-test="cve-issue-key" label="CVE ID" name="key" required />
         </StackItem>
         <StackItem>
           <ComponentField name="components" />
         </StackItem>
-        <StackItem>
-          <InputField data-test="cve-url" label="URL for the CVE" name="url" required />
```

Comment:
We would still need this field just renamed to source, right?

Reply:
No, As per the new designs, we are ommitting source and adding packages field